_posts/en/2021-02-19-pausing-memberships.md

```diff
@@ -0,0 +1,80 @@
+---
+title: 'Pausing the Membership Program'
+permalink: /en/pausing-memberships/
+name: 2021-02-19-pausing-memberships
+slug: 2021-02-19-pausing-memberships
+type: posts
+layout: post
+lang: en
+version: 1
+excerpt: >
+  As business travel remains on hold in many parts of the world due to the
+  COVID-19 pandemic, we're temporarily pausing our membership model. We'll
+  continue to provide high-quality online technical material for engineers
+  working on Bitcoin.
+---
+
+As business travel remains on hold in many parts of the world due to the
+COVID-19 pandemic, we're temporarily pausing our membership model. We'll
+continue to provide high-quality online technical material for engineers
+working on Bitcoin.
+
+We [launched Bitcoin Optech][Optech launch] in July 2018 with a very clear
+mission: to help Bitcoin companies adopt scaling technologies and to raise
+the level of conversation around scaling Bitcoin. Over the previous months,
+the Bitcoin community had been through the bitter and contentious 'fork wars',
+and our ethos was (and remains) to focus on the technology and provide
+sound technical guidance for anyone wishing to use the Bitcoin block chain in
+a more efficient way.
+
+Our work is made possible by very generous founding donations from John
+Pfeffer, Wences Casares and Chaincode Labs, and ongoing funding from our
```

Comment:
Nit: suggest changing the second "and" to "with".

Reply:
I agree that's better. Thanks!